tasks/selinux_load_module.yml

```diff
@@ -0,0 +1,45 @@
+- name: Set defaults
+  set_fact:
+    name: "{{ item.name | default('')}}"
+    state: "{{ item.state | default('enabled') }}"
+    priority: "{{ item.priority | default('400') }}"
+    store: "{{ selinux_policy | default(ansible_selinux.type, true) }}"
+
+- name: Find installed module
+  set_fact:
+    installed_module: "{{ lookup('selinux_modules', name, priority=priority) | default({ 'name': '', 'priority': '', 'state': 'absent' }, true)}}"
+
+- name: Install module
+  block:
+    - name: Create temporary directory
+      tempfile:
+        state: directory
+        prefix: linux_system_role.selinux
+      register: tempdir
+
+    - name: Copy module file "{{ item.path }}"
+      copy:
+        dest: "{{ tempdir.path }}"
+        src: "{{ item.path }}"
+
+    - name: Install module
+      command: "semodule -N -s {{ store }} -X {{ priority }} -i {{ tempdir.path }}/{{ item.path | basename }}"
+
```

Comment:
The clean up of the temporary directory should be done in a `rescue` block

Reply:
https://github.com/linux-system-roles/selinux/pull/56/commits/c139eeab3791bf85e9155b017c066b9d661730ea